.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,103 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bugs
+description: Report a bug with the Microsoft build of Go
+title: "import/path: issue title"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve! 🙏 Please answer these questions and provide as much information as possible about your problem.
+
+  - type: input
+    id: go-version
+    attributes:
+      label: Microsoft build of Go version
+      description: |
+        What version of Go are you using (`go version`)?
+
+        Note: we only [support](https://github.com/microsoft/go/blob/microsoft/main/eng/doc/Downloads.md) the two most recent major releases.
+      placeholder: ex. go version go1.24.1 darwin/arm64
+    validations:
+      required: true
+
+  - type: textarea
+    id: platform
+    attributes:
+      label: What is your operating system and platform?
+      description: Please add as much information here as possible including your Operating System, architecture etc.
+      placeholder: e.g "Ubuntu 24.04 on ARM64" or "Windows 10 2004 on x86-64"
```

```suggestion
      placeholder: e.g. "Ubuntu 24.04 on ARM64" or "Windows 10 2004 on x86-64"
```

(Or switch to `ex.` like the above?)